packages/contract-artifacts/src/index.ts

```diff
@@ -37,6 +38,7 @@ export {
     ERC1155Mintable,
     ERC1155Proxy,
     ERC20BridgeProxy,
+    ERC20BridgeSampler,
```

Comment:
Can't whoever just import this and the wrapper from `@0x/asset-swapper`?

Reply:
Defeats the whole point of bundling the contracts since we have to gitpkg those two again. :cry: